_rules/video-transcript-1a02b0.md

```diff
@@ -129,6 +148,20 @@ A video element with a link to an incorrect text transcript on a different page.
 </html>
 ```
 
+#### Failed Example 3
+This `video` element has a text transcript available on the same page, but the transcript is not [included in the accessibility tree][].
+```html
+<html lang="en">
+<video controls>
+  <source src="/test-assets/rabbit-video/video.mp4" type="video/mp4"></source>
+  <source src="/test-assets/rabbit-video/video.webm" type="video/webm"></source>
+</video>
+<p aria-hidden="true">The above video shows a giant fat rabbit climbing out of a hole in the ground.
+He stretches, yaws, and then starts walking.
+Then he stops to scratch his bottom.</p>
+</html>
+```
+
```

Comment:
This collides with the changes made in #2064.

Reply:
Thanks for catching that. Requested more details in #2186

Reply:
This will be a passed example since a transcript is available. [This PR](https://github.com/act-rules/act-rules.github.io/pull/2064/files#diff-41eff5772818db1bedbdd0cbb88ffa9fc1447434070c9a384ca375da9b5560b1) is removing "included in the accessibility tree" from the expectation.